integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporterWebApp.java

```diff
@@ -334,7 +333,7 @@ void testAdminPortEnabled() throws Exception {
           true, null),
           "monitoring exporter metrics page can't be accessed via https");
     } finally {
-      shutdownDomain(domain2Namespace, domain2Uid);
+      //shutdownDomain(domain2Uid, domain2Namespace);
```

Comment:
if you don't want to shutdown domain here, remove this line. Otherwise remove "//"

Reply:
fixed